vite.preload.config.ts

```diff
@@ -21,8 +22,16 @@ export default defineConfig((env) => {
           assetFileNames: '[name].[ext]',
         },
       },
+      sourcemap: true,
```

Comment:
I also noticed that sourcemaps were configured to be uploaded only for the `main` process so I added the same config for `preload` and `renderer` as well.

https://github.com/grafana/k6-studio/blob/c73eb40ff012c99fe190cd7c916de7bb89e2d134/vite.main.config.ts#L26